packages/components/src/ellipsis-menu/index.js

```diff
@@ -48,6 +48,7 @@ class EllipsisMenu extends Component {
 			<div className="woocommerce-ellipsis-menu">
 				<Dropdown
 					contentClassName="woocommerce-ellipsis-menu__popover"
+					focusOnMount="container"
```

Comment:
From an accessibility standpoint, does this makes sense?

Reply:
I'm not sure, and probably not since it's overriding the default behavior of `Dropdown` just to handle a visual treatment. I'll revert this change before merging.